tests/python/unittest/test_tvmscript_roundtrip.py

```diff
@@ -2961,15 +2961,15 @@ def test_abs():
 
 
 @tvm.script.tir
-def printer(a: ty.handle) -> None:
+def constant_folding(a: ty.handle) -> None:
     A = tir.match_buffer(a, (), "float32")
     A[()] = tir.min(2.2, 5.2)
-    A[()] = tir.max(tir.float32(2.2), tir.float32(5.2))
+    A[()] = tir.max(tir.float32(2.2), tir.float32(tir.float32(5.2)))
```

Comment:
`tir.float32(tir.float32(5.2)))` seems strange to me... Is it indented? 🤔

Reply:
it is intended to test the fact that we actually fold conversions of float32 into one